Add new influxdb3 write and query input methods (#5807)
influxdata/docs-v2: content/shared/influxdb3-cli/write.md

<!--pytest.mark.skip-->

```bash
influxdb3 write [OPTIONS] --database <DATABASE_NAME> [LINE_PROTOCOL]...
```

##### Aliases

`write`, `w`

## Arguments

- **LINE_PROTOCOL**: The line protocol to write to {{< product-name >}}.
  Provide the line protocol in one of the following ways:

>   - a string on the command line

Reviewer: ```suggestion
  - a string
```

  - a path to a file that contains the line protocol using the `--file` option
  - from stdin

## Options

| Option |                    | Description                                                                              |
| :----- | :----------------- | :--------------------------------------------------------------------------------------- |
| `-H`   | `--host`           | Host URL of the running {{< product-name >}} server (default is `http://127.0.0.1:8181`) |
| `-d`   | `--database`       | _({{< req >}})_ Name of the database to operate on                                       |
|        | `--token`          | Authentication token                                                                     |

> | `-f`   | `--file`           | Line protocol file to use to write data                                                  |

Reviewer: ```suggestion
| `-f`   | `--file`           | A file that contains line protocol to write                                                 |
```
